### PR TITLE
Updating demo utilities to support creating Certificates

### PR DESCRIPTION
**Pull request by PeterHamilton**
This change updates the unit demo utilities, adding in support for the creation, registration, and retrieval of X.509 Certificate objects. Support includes an example encoding of a Certificate as well as updated logging routines to display retrieved Certificate data.

### CODE REVIEW

File: kmip/demos/utils.py [hunk 1]
```diff
@@ -179,8 +183,11 @@ def build_cli_parser(operation):
 
 
 def build_cryptographic_usage_mask(logger, object_type):
-    if (object_type == ObjectType.SYMMETRIC_KEY or
-       object_type == ObjectType.SECRET_DATA):
+    if object_type == ObjectType.CERTIFICATE:
+        flags = [CryptographicUsageMask.SIGN,
```
Comment (rellerreller) [inline kmip/demos/utils.py]: This should be encrypt and verify.

File: kmip/demos/utils.py [hunk 2]
```diff
@@ -255,9 +273,61 @@ def build_cryptographic_algorithm(logger, object_type):
         sys.exit()
 
 
-def build_key_value(logger, object_type):
-    if (object_type == ObjectType.SYMMETRIC_KEY
-       or object_type == ObjectType.SECRET_DATA):
+def build_secret_value(logger, object_type):
```
Comment (rellerreller) [inline kmip/demos/utils.py]: Can you add a note on how you generated this like for the public and private key?

Comment (PeterHamilton) [inline kmip/demos/utils.py]: Yep, should've done that earlier.

File: kmip/demos/utils.py (hunk 1 above)
Comment (PeterHamilton) [inline kmip/demos/utils.py]: Ok, I'll change it and run a test to make sure it works.

**Comment by PeterHamilton** [comment]
@rellerreller Ok, changes made.

**Comment by rellerreller** [comment]
:+1:

**Pull request merged**